CMakeLists.txt

```diff
@@ -163,7 +163,56 @@ file(MAKE_DIRECTORY "${scylla_gen_build_dir}")
 include(add_version_library)
 generate_scylla_version()
 
+option(Scylla_USE_PRECOMPILED_HEADER "Use precompiled header for Scylla" ON)
+add_library(scylla-precompiled-header INTERFACE)
+target_link_libraries(scylla-precompiled-header INTERFACE
+    absl::headers
+    absl::btree
+    absl::hash
+    absl::raw_hash_set
+    Seastar::seastar
+    Snappy::snappy
+    systemd
+    ZLIB::ZLIB
+    lz4::lz4_static
+    zstd::zstd_static)
+add_library(scylla-precompiled-header-impl STATIC exported_templates.cc)
```

Comment:
@radoslawcybulski I think you forgot to commit this `exported_templates.cc` thingy